src/database/sql/audio.py

```diff
@@ -434,7 +467,8 @@ def get_level(self, audio_id: str) -> str:
         query = """
             SELECT peak, Loudness_integrated, loudness_range, last_updated
             FROM Audio
-            WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+            WHERE audio_release=:rel AND audio_side=:side
+            AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.